packages/plugin-team/README.md

```diff
@@ -0,0 +1,150 @@
+# addConversation
```

Comment:
Follow up: it should be committed, but empty. with the no-autogen-readmes PR, you'll have some unfortunate conflicts. If I get that one merged before this gets merged, change this readme to be simple description and a pointer to the docs site.